bamboo/bootstrap-tf-deployment.sh

```diff
@@ -71,6 +71,7 @@ echo "terraform {
 ../terraform init \
   -input=false
 
+## TODO: This can be removed entirely once most stacks have run through Bamboo
```

Comment:
The updates in this PR result in the separate stack removing the lambda, security groups, etc.